#!/bin/bash
#
#SBATCH --job-name="Making expression files" # job name
#SBATCH --time=144:00:00 #maximum runtime
#SBATCH --mem=50G #maximum memmory allocation
#SBATCH --mail-user=julia.raices@ist.ac.at #send e-mail to user for everything
#SBATCH --mail-type=ALL
# loads module from which raw data should be dowloaded
export OMP_NUM_THREADS=5 # this makes it usefive threads

#srun --cpu_bind=verbose ./STAR/source/STAR --runThread 5 --genomeDir --readFilesIn --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts

## [mouse | ref 8] Gompers et al 2017
### RNA-seq Heterozygous (Het) P56 adult
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629564.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetP56Adult_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629565.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetP56Adult_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629566.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetP56Adult_3
### RNA-seq WT (Wild Type) P56 adult
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629561.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTP56Adult_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629562.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTP56Adult_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629563.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTP56Adult_3
### RNA-seq Het P0.Chd8 <- that's the one we want!
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629555.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetP0.Cdh8_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629556.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetP0.Cdh8_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629557.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetP0.Cdh8_3
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629558.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetP0.Cdh8_4
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629559.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetP0.Cdh8_5
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629560.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetP0.Cdh8_6
### RNA-seq WT P0.Chd8 <- that's the one we want!
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629550.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTP0.Cdh8_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629551.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTP0.Cdh8_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629552.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTP0.Cdh8_3
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629553.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTP0.Cdh8_4
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629554.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTP0.Cdh8_5
### RNA-seq Het e17.5
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629543.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetE17.5_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629544.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetE17.5_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629545.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetE17.5_3
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629546.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetE17.5_4
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629547.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetE17.5_5
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629548.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetE17.5_6
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629549.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetE17.5_7
### RNA-seq WT e17.5
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629540.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTE17.5_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629541.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTE17.5_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629542.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTE17.5_3
### RNA-seq Het e14.5
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629534.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetE14.5_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629535.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetE14.5_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629536.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetE14.5_3
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629537.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetE14.5_4
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629538.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetE14.5_5
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629539.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetE14.5_6
### RNA-seq WT e14.5
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629531.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTE14.5_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629532.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTE14.5_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629533.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTE14.5_3
### RNA-seq Het e12.5
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629527_1.fastq Gompers2017/SRR5629527_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetE12.5_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629528_1.fastq Gompers2017/SRR5629528_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetE12.5_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629529_1.fastq Gompers2017/SRR5629529_2.fastq--outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetE12.5_3
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629530_1.fastq Gompers2017/SRR5629530_2 --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersHetE12.5_4
### RNA-seq WT e12.5
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR1/SRR5629519.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTe12.5_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629520.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTe12.5_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629521.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTe12.5_3
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629522.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTe12.5_4
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629523.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTe12.5_5
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629524.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTe12.5_6
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629525.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTe12.5_7
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Gompers2017/SRR5629526.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./GompersWTe12.5_8
## [mouse | ref 10] Katayama et al 2016 -> theoretically it is available, but it's impossible to find 
## [mouse | ref 14] Nishiyama et al 2009 -> no available raw data (for more information Jules has a nice resume of all we have
## [mouse | ref 16] Platt et al 2017
### RNA-seq NAc WT
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350401_1.fastq Platt2017/SRR5350401_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTNAc_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350400_1.fastq Platt2017/SRR5350400_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTNAc_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350399_1.fastq Platt2017/SRR5350399_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTNAc_3
### RNA-seq NAc Het
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350398_1.fastq Platt2017/SRR5350398_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetNAc_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350397_1.fastq Platt2017/SRR5350397_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetNAc_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350396_1.fastq Platt2017/SRR5350396_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetNAc_3
### RNA-seq DS WT
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350395_1.fastq Platt2017/SRR5350395_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTDS_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350394_1.fastq Platt2017/SRR5350394_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTDS_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350393_1.fastq Platt2017/SRR5350393_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTDS_3
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350392_1.fastq Platt2017/SRR5350392_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTDS_4
### RNA-seq DS Het
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350391_1.fastq Platt2017/SRR5350391_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetDS_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350390_1.fastq Platt2017/SRR5350390_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetDS_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350389_1.fastq Platt2017/SRR5350389_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetDS_3
### RNA-seq CC WT
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350388_1.fastq Platt2017/SRR5350388_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTCC_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350387_1.fastq Platt2017/SRR5350387_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTCC_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350386_1.fastq Platt2017/SRR5350386_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTCC_3
### RNA-seq CC Het
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350385_1.fastq Platt2017/SRR5350385_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetCC_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350384_1.fastq Platt2017/SRR5350384_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetCC_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350383_1.fastq Platt2017/SRR5350383_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetCC_3
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350382_1.fastq Platt2017/SRR5350382_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetCC_4
### RNA-seq WT HPF
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350381_1.fastq Platt2017/SRR5350381_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTHPF_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350380_1.fastq Platt2017/SRR5350380_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTHPF_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350379_1.fastq Platt2017/SRR5350379_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTHPF_3
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350378_1.fastq Platt2017/SRR5350378_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTHPF_4
### RNA-seq Het HPF
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350377_1.fastq Platt2017/SRR5350377_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetHPF_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350376_1.fastq Platt2017/SRR5350376_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetHPF_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350375_1.fastq Platt2017/SRR5350375_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetHPF_3
### RNA-seq WT AM
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350374_1.fastq Platt2017/SRR5350374_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTAM_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350373_1.fastq Platt2017/SRR5350373_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTAM_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350372_1.fastq Platt2017/SRR5350372_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTAM_3
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350371_1.fastq Platt2017/SRR5350371_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTAM_4
### RNA-seq Het AM
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350370_1.fastq Platt2017/SRR5350370_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetAM_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350369_1.fastq Platt2017/SRR5350369_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetAM_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350368_1.fastq Platt2017/SRR5350368_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetAM_3
### RNA-seq WT LH
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350367_1.fastq Platt2017/SRR5350367_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTLH_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350366_1.fastq Platt2017/SRR5350366_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTLH_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350365_1.fastq Platt2017/SRR5350365_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTLH_3
### RNA-seq Het LH
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350364_1.fastq Platt2017/SRR5350364_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetLH_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350363_1.fastq Platt2017/SRR5350363_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetLH_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350362_1.fastq Platt2017/SRR5350362_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetLH_3
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350361_1.fastq Platt2017/SRR5350361_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetLH_4
### RNA-seq WT VTA (there's no replciate 1)
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350360_1.fastq Platt2017/SRR5350360_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTVTA_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350359_1.fastq Platt2017/SRR5350359_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTVTA_3
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350358_1.fastq Platt2017/SRR5350358_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattWTVTA_4
### RNA-seq Het VTA
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350357_1.fastq Platt2017/SRR5350357_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetVTA_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR5350356_1.fastq Platt2017/SRR5350356_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetVTA_2
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Mouse --readFilesIn Platt2017/SRR1/SRR5350355_1.fastq Platt2017/SRR1/SRR5350355_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./PlattHetVTA_3
## [human | ref 19] Wang et al 2017
### RNA-seq CDH8+/+ A
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Human --readFilesIn Wang2017/SRR1/SRR4017382_1.fastq Wang2017/SRR1/SRR4017382_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./WangWTCDH8A_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Human --readFilesIn Wang2017/SRR4017383_1.fastq Wang2017/SRR4017383_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./WangWTCDH8A_2
### RNA-seq CDH8+/+ B
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Human --readFilesIn Wang2017/SRR4017384_1.fastq Wang2017/SRR4017384_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./WangWTCDH8B_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Human --readFilesIn Wang2017/SRR4017385_1.fastq Wang2017/SRR4017385_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./WangWTCDH8B_2
### RNA-seq CDH8+/- A
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Human --readFilesIn Wang2017/SRR4017386_1.fastq Wang2017/SRR4017386_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./WangHetCDH8A_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Human --readFilesIn Wang2017/SRR4017387_1.fastq Wang2017/SRR4017387_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./WangHetCDH8A_2
### RNA-seq CDH8+/- B
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Human --readFilesIn Wang2017/SRR4017388_1.fastq Wang2017/SRR4017388_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./WangHetCDH8B_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Human --readFilesIn Wang2017/SRR4017389_1.fastq Wang2017/SRR4017389_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./WangHetCDH8B_2
### RNA-seq CDH8+/- C
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Human --readFilesIn Wang2017/SRR4017390_1.fastq Wang2017/SRR4017390_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./WangHetCDH8C_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Human --readFilesIn Wang2017/SRR4017391_1.fastq Wang2017/SRR4017391_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./WangHetCDH8C_2
### RNA-seq CDH8+/- D
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Human --readFilesIn Wang2017/SRR4017392_1.fastq Wang2017/SRR4017392_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./WangHetCDH8D_1
srun --cpu_bind=verbose ./STAR/source/STAR --runThreadN 5 --genomeDir Human --readFilesIn Wang2017/SRR4017393_1.fastq Wang2017/SRR4017393_2.fastq --outSAMtype BAM SortedByCoordinate --alignIntronMax 100000 --outFilterIntronMotifs RemoveNoncanonical --outFilterMultimapNmax 1 --quantMode GeneCounts --outFileNamePrefix ./WangHetCDH8D_2



